assets/components/NotificationList.tsx

```diff
@@ -70,9 +85,11 @@ class NotificationList extends React.Component<any, any> {
     }
 
     render() {
+        const notificationArePaused: boolean = new Date(this.props.fullUser.notification_schedule?.pauseFrom ?? '') < new Date();
```

Comment:
You should't be creating a date from an empty string. If it's not defined, simply set `notificationArePaused` to `false` and compare dates only if it's defined.